backend/src/graphql/queries/order.js

```diff
@@ -35,3 +38,64 @@ export const orderByIdOfUser = schemaComposer
     },
   })
   .wrapResolve(requiredAuth);
+
+const orderSummaryPayload = schemaComposer.createObjectTC({
+  name: 'orderSummaryPayload',
+  fields: {
+    x: 'String',
+    y: 'Float',
+  },
+});
+export const orderSummary = schemaComposer
+  .createResolver({
+    name: 'orderSummary',
+    kind: 'query',
+    type: [orderSummaryPayload],
+    resolve: async ({ context }) => {
+      // const { role } = context.user;
+      // if (role !== 'admin') {
+      //   throw new ValidationError('โปรดใช้บัญชีแอดมินในการเข้าใช้งาน');
+      // }
```

Comment:
@gearprn, It seem have a report that `context.user` don't have a role, any idea?